spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+require "users_controller"
+
+describe UsersController do
```

Comment:
A suggestion to prefix this describe block with `RSpec.`

Ref: https://stackoverflow.com/questions/24747448/rspec-describe-vs-describe